dietpi/dietpi-services

```diff
@@ -272,6 +272,7 @@ _EOF_
 
 		# Additional services: https://github.com/MichaIng/DietPi/issues/1869#issuecomment-401017251
 		[[ -f '/etc/rsyncd.conf' ]] && aSERVICE_NAME+=('rsync')
+		[[ -d '/opt/adsb' ]] && aSERVICE_NAME+=('adsb-docker' 'adsb-setup')
```

Comment:
These can be added to the array above below YaCy. It is internally checked for the existence of those services and I guess the names are sufficiently specific that we do not need to check for the existence of the install dir as well.

Reply:
Excellent. I wasn't sure what the preferred way was for doing that as I saw how rsync was added there... I'll make that change.